assets/src/util/calendar-utils.js

```diff
@@ -226,3 +228,40 @@ export function adjustAsyncResourcesBusinessHours(resources, calendarRef, date)
 
   return false;
 }
+
+/**
+ * RemoveDuplicateEvents - Checks if any events stored in the internal storage of events loaded asynchronously are
+ * dupliactes, and removes them.
+ *
+ * @param {object} internalAsyncEvents Object of all resource events gathered async
+ * @returns {object} Object of all resource events gatered async, cleaned for duplicates
+ */
+export function removeDuplicateEvents(internalAsyncEvents) {
+  internalAsyncEvents.forEach((ev1, ix1) => {
```

Comment:
Instead of shortning parameter names, prefer full name. 

```suggestion
  internalAsyncEvents.forEach((event1, index1) => {
```